docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -15,6 +15,9 @@ There you can find links to upgrade notes for other versions too.
     - dump translations using `php phing dump-translations` and fill in the translations based on the changes from pull request
 - check whether you extended class or method `ImageFacade::copyImages` or used it in your project and make sure it works like you intended ([#851](https://github.com/shopsys/shopsys/pull/851))
     - *(low priority)* remove `/var/www/html/var/cache` folder from your `@main_filesystem` filesystem storage if exists, set as local filesystem storage in path `%kernel.project_dir%` by default
+- fixed files upload ([#869](https://github.com/shopsys/shopsys/pull/869))
```

Comment:
it's not an instruction I would drop this whole line and put PR link to next line.